src/h5web/dataset-visualizer/vis/MatrixVis.module.css

```diff
@@ -13,3 +13,37 @@
 .cell[data-bg] {
   background-color: var(--secondary-light-bg);
 }
+
+.indexCell {
+  display: flex;
+  font-weight: bold;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--primary-bg);
```

Comment:
Just some visual suggestions, take it or leave it:

- `font-weight: 600`
- alternating the background colour like what we do for data cells (this tool might help finding a lighter or darker colour: https://calcolor.co/blend/f5fbef/ffffff)
- making the background colour very slightly transparent with `rgba`

Reply:
First two implemented. I find that having a transparent background makes the numbers appear cluttered...

Reply:
Yeah, I wasn't entirely convinced either.